ckanext/datagovuk/public/datagovuk.css

```diff
@@ -110,3 +102,103 @@ p.home-footer-text {
   line-height: 1.5;
 }
 
+.form-control {
+  margin-bottom: 14px;
+}
+
+.mt-10 {
+  margin-top: 10px;
+}
+
+.mb-10 {
+  margin-bottom: 10px;
+}
+
+.control-label.col-md-3 {
+  text-align: right;
+  padding-right: 0;
+}
+
+.control-full input[type=radio]{
+  width: auto;
+}
+
+.harvest-types label.radio {
+  margin-bottom: 0;
+}
+
+.harvest-types label.radio input{
+  top: -1px;
+}
+
+.editor-info-block {
+  margin-top: -14px;
+  margin-bottom: 14px;
+}
+
+.float-container,
+.form-group {
+  overflow: auto;
+}
+
+.form-group {
+  margin-bottom: 14px;
+}
+
+.form-control {
+  height: 32px;
+}
+
+.select2-container.form-control {
+  padding: 0;
+  border: 0;
+}
+
+.select2-drop {
+  margin-top: -7px;
```

Comment:
I don't know if we need this. I tested without it and it seems to work perfectly fine without it.

Reply:
This is a fix for this:
![Screenshot 2020-05-20 at 11 00 20](https://user-images.githubusercontent.com/31649453/82433536-7c032780-9a89-11ea-964e-d70dc6973cf5.png)